### PR TITLE
OSDOCS#45783: Added a Release Note on CentOS 7+8 support removal

### PR DESCRIPTION
**Pull request by jherrman**
Version(s):
4.15

Issue:
https://issues.redhat.com/browse/CNV-45783

Link to docs preview:
https://84705--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/release_notes/virt-4-15-release-notes.html

QE review:
- [x] QE has approved this change.

Additional information:
Sort-of-clone of https://github.com/openshift/openshift-docs/pull/84704

### CODE REVIEW

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -1513,6 +1513,7 @@ To view information about the integrated {product-registry}, run `oc get imagest
 
 As of {product-title} {product-version}, support for installing clusters on {rh-openstack} with kuryr is removed.
 
+
```
Comment (RoniKishner) [inline release_notes/ocp-4-15-release-notes.adoc:1516]: space by mistake?

Comment (jherrman) [reply release_notes/ocp-4-15-release-notes.adoc:1516]: Yes, thank you for catching it

File: virt/release_notes/virt-4-15-release-notes.adoc [hunk 2]
```diff
@@ -187,6 +187,9 @@ Removed features are not supported in the current release.
 //Leaving this as is for 4.15
 * Support for the legacy HPP custom resource, and the associated storage class, has been removed for all new deployments. In {VirtProductName} {VirtVersion}, the HPP Operator uses the Kubernetes Container Storage Interface (CSI) driver to configure local storage. A legacy HPP custom resource is supported only if it had been installed on a previous version of {VirtProductName}.
 
+//CNV-45783
+* CentOS 7 and CentOS Stream 8 are now in the End of Life phase. As a consequence, the container ímages for these operating systems have been removed from OpenShift Virtualization and are no longer link:https://access.redhat.com/articles/4234591#community[community supported].
```
Comment (skopacz1) [inline virt/release_notes/virt-4-15-release-notes.adoc:191]:
```suggestion
* CentOS 7 and CentOS Stream 8 are now in the End of Life phase. As a consequence, the container images for these operating systems have been removed from OpenShift Virtualization and are no longer link:https://access.redhat.com/articles/4234591#community[community supported].
```

**Review by skopacz1 — COMMENTED** [review]
Same comment as for the others

**Pull request merged**